test/test_palo_alto_collectors.py

```diff
@@ -2,26 +2,28 @@
 
 import os
 import sys
+from dotenv import load_dotenv
 
 # Change to the Net-Manage repository so imports will work
-nm_path = os.environ.get('NM_PATH')
-os.chdir(f'{nm_path}/test')
+load_dotenv()
+netmanage_path = os.environ['netmanage_path']
+os.chdir(f'{netmanage_path}/test')
 sys.path.append('..')
```

Comment:
This should work without lines 10 and 11.  They shouldn't be needed since we made NM installable as a package.

Reply:
Updated and removed lines.